lib/matplotlib/text.py

```diff
@@ -823,10 +814,10 @@ def set_backgroundcolor(self, color):
 
         ACCEPTS: any matplotlib color
         """
-        if self._bbox is None:
-            self._bbox = dict(facecolor=color, edgecolor=color)
+        if self._bbox_patch is None:
+            self.set_bbox = dict(facecolor=color, edgecolor=color)
         else:
-            self._bbox.update(dict(facecolor=color))
+            self._bbox_patch.update(dict(facecolor=color))
```

Comment:
Why?  As it is, the update of the existing _bbox_patch is needed only if one hasn't just created a new one with the desired color.